app/views/devise/passwords/new.html.erb

```diff
@@ -14,7 +14,7 @@
 <%= validated_form_for(@password_reset_email_form,
                        url: user_password_path,
                        html: { autocomplete: 'off', method: :post, role: 'form' }) do |f| %>
-  <%= f.input :email, required: true, input_html: { 'aria-describedby': 'email-description' } %>
+  <%= f.input :email, required: true, input_html: { aria: { invalid: true, describedby: 'email-description' } } %>
```

Comment:
For some reason, doing `'aria-invalid': true` just never rendered to the DOM, so I had to do this syntax, similar to how `data-` prefixed keys work

Reply:
Curious if `'aria-invalid': 'true'` would have worked (i.e. maybe boolean values only treated specially for `aria` and `data` groupings?).

Reply:
No dice! That did not seem to work. I have a *feeling* that this is due to how the `input_html` is passed through various translation layers to ActionView, but haven't dug deep enough to confirm

Reply:
🤷  IMO, the object form seems quite nice as it is anyhow.